Updated documentation and Csv

MirzaBaig715 opened the pull request:
@sadanarshad Added documentation and correct errors.File Handling with CSV.

MirzaBaig715 committed:
Updated docs

Added documentation and correct errors

sadan commented on weatherman.py:
```diff
@@ -1,214 +1,362 @@
+                        )
+                  )
+# Take input from user. What type of report to display
+path = "*.txt"
```
What is this? This is not complete path of the weather files i guess

MirzaBaig715 replied:
files are in current Project Directory. We just have to extract _*.txt_ files from folder.

sadan commented on weatherman.py:
```diff
@@ -1,214 +1,362 @@
+            f.__next__()
+            f.__next__()
+            # Read all from a file and store it in List @lines
+            lines = csv.reader(f)
```
use CSV dict reader to extract data from files. That will refactor and improve your logic more.

MirzaBaig715 replied:
Yes i will extract using DictReader. Will also make it more memory efficient. Thanks for reviewing.

sadan commented on weatherman.py:
```diff
@@ -1,214 +1,362 @@
+    """
+
+    print(str("Year").rjust(7),
+          str("MAXTemp").rjust(7),
```
this is good.

MirzaBaig715 replied:
That's Great ! Thanks

MirzaBaig715 committed:
Csv and Memory efficient Code

Used Csv and i have made a alot of changes to make it more memory
efficient. Documentation is also written.

sadan commented on weatherman.py:
```diff
@@ -1,214 +1,206 @@
+    """
+    files = glob.glob("*.txt")
+    for file in files:
+        MAXTemp = []
```
variable name are always written in small letters **max_temp**, unless some sort of global or settings variable written in all caps.

MirzaBaig715 committed:
Variables names changed

corrected local variables name
